src/atoms/Button/Button.tsx

```diff
@@ -7,16 +7,18 @@ import Typography from '../Typography';
 
 interface Props {
   large?: boolean;
+  secondary?: boolean;
 }
 
 export const Button = styled(Typography)<Props>`
-  background: ${props => props.theme.primary};
-  border: none;
+  background: ${props => (props.secondary ? 'none' : props.theme.primary)};
+  border: ${props =>
+    props.secondary ? '1px solid' + props.theme.primary : 'none'};
```

Comment:
Instead of using `none` in a ternary, I prefer to do something like `${props => !props.secondary && props.theme.primary}`